docs/docusaurus/docs/cloud/connect/connect_databrickssql.md

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_label: 'Connect GX Cloud to Databricks SQL'
+title: 'Connect GX Cloud to Databricks SQL'
+description: Connect GX Cloud to a Databricks SQL Data Source.
+---
+
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+
+## Prerequisites
+
+- You have a [GX Cloud account](https://greatexpectations.io/cloud) with [Admin or Editor permissions](../about_gx.md#roles-and-responsibilities).
+
+- You have a Databricks SQL catalog, schema, and table.
+
+- To improve data security, GX recommends creating a separate Databricks SQL user for your GX Cloud connection.
```

Comment:
`s/user/service principal/` - Per Databricks [documentation](https://docs.databricks.com/en/admin/users-groups/service-principals.html), a service principal, not a user, should be used for automated tools.